src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -369,13 +369,15 @@ public void setMigratedCaseLinkDetails(String authToken, CaseDetails caseDetails
         // elastic search query
         List<SubmitEvent> submitEvent = caseRetrievalForCaseWorkerService.transferSourceCaseRetrievalESRequest(
                 caseDetails.getCaseId(), authToken, caseTypeIdsToCheck);
+        log.info("SubmitEvent is retrieved from ES for the update target case: {}.");
```

Comment:
@btensay are we logging something else out here? the {} doesn't have anything set to it. maybe a size check or get(0).getCaseId()

Reply:
yeah I was meant to add the case id. fixed it .